pages/index.vue

```diff
@@ -1,9 +1,9 @@
 <template>
     <div class="relative">
         <div class="flex flex-col h-full bg-dark">
-            <Header class="mb-10 || md:mb-20"/>
-            <Hero class="mb-10 || lg:mb-14 || 2xl:mb-20"/>
-            <Steps class="z-20"/>
+            <Header class="mb-10 || md:mb-20" />
+            <Hero class="mb-10 || lg:mb-14 || 2xl:mb-20" />
+            <Steps class="z-20" />
```

Comment:
For what are these `||` ?
What are they doing?

Reply:
They do not have a code-functionality.
Their purpose is for me, to have better readablity between different screen-sizes. 

So whenever these "| |" are in the code, their is a switch to the next larger, screen size!

Reply:
It is not commen practice to have these because if you may have large class sections it could be more confusing then help full.
If you want to keep track it's better to group them after each to see them together.